mbed_flasher/common.py

```diff
@@ -56,10 +56,15 @@ def check_points_unchanged(self, target):
         """
         new_target = {}
         if platform.system() == 'Windows':
-            mbeds = mbed_lstools.create()
-            if target['serial_port'] != mbeds.get_mbed_com_port(target['target_id']):
-                new_target['serial_port'] = mbeds.get_mbed_com_port(target['target_id'])
-
+            mbedls = mbed_lstools.create()
+            mbeds = mbedls.list_mbeds(filter_function=
+                                      lambda m: m['target_id'] == target['target_id'])
+            if len(mbeds) == 1:
+                if target['serial_port'] != mbeds[0]['serial_port']:
+                    new_target['serial_port'] = mbeds[0]['serial_port']
```

Comment:
You really can just set `new_target` to `mbeds[0]` if you want. I'd suggest doing that instead of doing selective updates.

That is of course you're doing something strange that requires the selective update 😛

Reply:
I think we have to do other refactoring as well - so lets keep this for now..